### PR TITLE
IIB Release v8.1.0

### PR DESCRIPTION
**Pull request by yashvardhannanavati**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+## 8.1.0
```
Comment (zanssa) [inline CHANGELOG.md:9]: Can we follow the same logic of the last release by adding the commit and the PR #?

Comment (chandwanitulsi) [reply CHANGELOG.md:9]: +1, that would be an improvement.
I am good with approving the PR as that shouldn't block the PR IMO

**Review by chandwanitulsi — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (yashvardhannanavati) [reply CHANGELOG.md:9]: Done

Comment (zanssa) [reply CHANGELOG.md:9]: Thank you!

**Review by zanssa — APPROVED** [review]

**Pull request merged**